meta-lmp-base/recipes-bsp/lmp-boot-firmware/lmp-boot-firmware.bb

```diff
@@ -54,16 +54,15 @@ do_install() {
             for file in `ls ${D}${nonarch_base_libdir}/${FIRMWARE_DEPLOY_DIR}/`; do
                 version="${version}-`md5sum ${D}${nonarch_base_libdir}/${FIRMWARE_DEPLOY_DIR}/${file} | cut -d' ' -f1`"
             done
-
-	    # limit the length of version
-	    version="`echo ${version} | md5sum | cut -d' ' -f1`"
+            # limit the length of version
+            version="`echo ${version} | md5sum | cut -d' ' -f1`"
```

Comment:
nit, can you drop these changes or isolate them in another patch (tab x spaces)?

Reply:
done